docs/operation/volume_management/volume_deletion.rst

```diff
@@ -0,0 +1,48 @@
+Removing Volumes
+================
+
+This section highlights how to delete a **Volume** in a MetalK8s cluster.
+
+Requirements
+------------
+
+A volume can be removed under the following conditions:
+
+- There is no backing storage.
+- The volume is not in use.
+
+Otherwise, the volume will be marked for deletion and remains
```

Comment:
Until one of these conditions occurs, the volume remains available but is marked for deletion.